configs/foreman/3.0.yaml

```
@@ -0,0 +1,543 @@
+---
+:project: foreman
+:github_org: theforeman
+:releases:
+  :3.0.0:
+    :redmine_version_id: 1367
```

Shouldn't this point to https://projects.theforeman.org/versions/1408?
```suggestion
    :redmine_version_id: 1408
```

---

It should.  Fixed.